pages/people/index.tsx

```diff
@@ -1,10 +1,20 @@
 import Nav from '@hashicorp/react-nav'
 import Footer from '@hashicorp/react-footer'
+import rivetQuery from '@hashicorp/nextjs-scripts/dato/client'
+import { GetStaticPropsResult } from 'next'
+import { PersonRecord, DepartmentRecord } from 'types'
 import style from './style.module.css'
 import query from './query.graphql'
-import rivetQuery from '@hashicorp/nextjs-scripts/dato/client'
 
-export default function PeoplePage({ allPeople, allDepartments }) {
+interface Props {
+  allPeople: PersonRecord[]
+  allDepartments: DepartmentRecord[]
+}
+
+export default function PeoplePage({
+  allPeople,
+  allDepartments,
+}: Props): React.ReactElement {
```

Comment:
Kinda nit, but typing the return type of a component like this seems unnecessary. We never consume a page and care about its return value directly.

Reply:
Agreed - I'd like to change this in our default eslint config though if that's where we want to go 😀